src/stan/services/util/experimental_message.hpp

```diff
@@ -0,0 +1,33 @@
+#ifndef STAN_SERVICES_UTIL_EXPERIMENTAL_MESSAGE_HPP
+#define STAN_SERVICES_UTIL_EXPERIMENTAL_MESSAGE_HPP
+
+#include <stan/callbacks/writer.hpp>
+
+namespace stan {
+  namespace services {
+    namespace util {
+
+      /**
+       * Writes an experimental message to the writer.
+       * All experimental algorithms should call this function.
+       *
+       * @param[in,out] message_writer writer for experimental algorithm message
+       */
+      void experimental_message(stan::callbacks::writer&
+                                message_writer) {
+        message_writer("------------------------------"
+                       "------------------------------");
+        message_writer("EXPERIMENTAL ALGORITHM:");
+        message_writer("  - please expect frequent updates to the procedure");
+        message_writer("  - please expect unexpected inference results");
```

Comment:
expect the unexpected?  how about instead of both lines:  

This procedure has not been thoroughly tested and may be unstable or buggy.  The interface is subject to change.